web/app/dao/SearchDAO.java

```diff
@@ -147,15 +147,14 @@
 			"ORDER BY 2 LIMIT ?, ?;";
 
 	public final static String SEARCH_AUTOCOMPLETE_LIST = "searchSource";
-
-	public final static String GET_METRIC_AUTO_COMPLETE_LIST = "SELECT DISTINCT CASE " +
-			"WHEN parent_path is null or parent_path = '' THEN field_name " +
-			"ELSE CONCAT_WS('.', parent_path,  field_name) END as full_name FROM dict_field_detail";
+	public final static String SEARCH_AUTOCOMPLETE_LIST_DATASET = "searchSourceDataset"; // check seyi
```

Comment:
@alyiwang removed. @theseyi probably for your reference, and log issue to me if integration testing fails